docs/getting-started.md

```diff
@@ -112,3 +112,27 @@ The SMTP_URL takes any format that that [Nodemailer](https://nodemailer.com/smtp
 
 
 [oauth2-jwt]: https://tools.ietf.org/html/draft-ietf-oauth-access-token-jwt-12
+
+
+Server Settings & Defaults
```

Comment:
Lets put this in its own document.

Some suggested flavor text:

The following table lists all configurable options for a12nserver. These can be changed in the `server_settings` table in your database.

The value for each field is stored as a JSON string. This means that in order to store `null`, you need the literal string `null`. To store a string, it needs to be surrounded by double-quotes. For example, a legal value for the `smpt.emailFrom` field might be `"info@example.org"`